core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -830,12 +836,42 @@ export class LoggedOutUser extends BaseUser {
       );
     } else {
       showMessage(
-        'The dismiss button closes the Thanks for Donating popup ' +
+        'The dismiss button closes the Thanks for Donating modal ' +
           'and shows the Donate page.'
       );
     }
   }
 
+  /**
+   * Function to click the dismiss button on the Thanks for Donating modal,
+   * and check if the Thanks for Donating modal disappears
+   * and if the About page is shown.
+   */
+  async dismissDonationThanksModalOnAboutPage(): Promise<void> {
+    await this.clickOn(dismissButton);
+    await this.page.waitForSelector(thanksForDonatingClass, {hidden: true});
+    const thanksForDonatingHeader = await this.page.$(thanksForDonatingClass);
+    if (thanksForDonatingHeader !== null) {
+      throw new Error(
+        'The dismiss button does not close the Thanks for Donating modal!'
+      );
+    }
+
+    await this.page.waitForSelector(aboutPage);
+    const donatePageShowed = await this.page.$(aboutPage);
+    if (donatePageShowed === null) {
+      throw new Error(
+        `The dismiss button should show the About page,
+          but it opens ${this.page.url()} instead.`
+      );
+    } else {
+      showMessage(
+        'The dismiss button closes the Thanks for Donating modal ' +
```

Comment:
Donation thanks modal, make sure to replace all the instances of "thanks for donating modal" with "donation thanks modal" for clarity and consistency.